### PR TITLE
Update _cleaning.erb

### PR DESCRIPTION
**Pull request by DGarland5**
:warning: Only merge changes if you are happy for them to go live. :warning: 

This application is now [continuously deployed](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request). Merged changes are automatically deployed to staging and production.

### CODE REVIEW

File: lib/smart_answer_flows/coronavirus-business-reopening/outcomes/_cleaning.erb [hunk 1]
```diff
@@ -11,6 +11,10 @@ To minimise the risk of the virus spreading you should:
 - provide more bins and empty them more often
 - clear workspaces and remove waste and belongings from the area at the end of a shift
 - use different gowns and towels for each client
+- make sure that housekeeping staff follow government handwashing guidelines, and make a checklist of all surfaces to be cleaned when each guest vacates
```
Comment (theseanything) [inline lib/smart_answer_flows/coronavirus-business-reopening/outcomes/_cleaning.erb:14]: Should we link to the handwashing guidance?

File: lib/smart_answer_flows/coronavirus-business-reopening/outcomes/_cleaning.erb [hunk 2]
```diff
@@ -36,64 +40,24 @@ There may be a risk of the virus coming into the workplace through goods, mercha
 - clean things like reusable delivery boxes regularly
 - minimise client contact with testers
 - sanitise all hand tools, controls, machinery and equipment after use
-- clean anything that’s touched regularly, such as buckets, site equipment
+- frequently clean anything that’s touched regularly, such as buckets, site equipment, door handles, pump handles and printers
```
Comment (theseanything) [inline lib/smart_answer_flows/coronavirus-business-reopening/outcomes/_cleaning.erb:43]: brackets for the example?

Comment (theseanything) [reply lib/smart_answer_flows/coronavirus-business-reopening/outcomes/_cleaning.erb:43]:
```suggestion
- frequently clean anything that’s touched regularly (such as buckets, site equipment, door handles, pump handles and printers)
```

File: lib/smart_answer_flows/coronavirus-business-reopening/outcomes/_cleaning.erb (hunk 1 above)
Comment (DGarland5) [reply lib/smart_answer_flows/coronavirus-business-reopening/outcomes/_cleaning.erb:14]: not sure that there is a link specifically for that?

Comment (DGarland5) [reply lib/smart_answer_flows/coronavirus-business-reopening/outcomes/_cleaning.erb:14]: I think it's the wash your hands for 20 seconds, thoroughly etc.

Comment (theseanything) [reply lib/smart_answer_flows/coronavirus-business-reopening/outcomes/_cleaning.erb:14]: No worries - if there''s not a clear link to add now we can, look at this in the future if its a problem

**Pull request merged**